### PR TITLE
Fix changes to CTA buttons with helper

### PR DESCRIPTION
**Pull request by mysteryihs**
I'm making a new PR, because I can't figure out how to change the old PR 👍

### CODE REVIEW

File: app/helpers/button_helper.rb [hunk 1]
```diff
@@ -3,6 +3,10 @@ def sign_up_button
     link_to 'Sign Up', sign_up_path(ref: 'success_stories'), class: 'button button--primary'
   end
 
+  def curriculum_button
+  	link_to 'Curriculum', curriculum_path(ref: 'success_stories'), class: 'button button--primary'
```
Comment (arun1595) [inline app/helpers/button_helper.rb]: Why is the referrer `success_stories`? :thinking:

Comment (mysteryihs) [inline app/helpers/button_helper.rb]: Oops, I just copy and pasted it, want me to remove it?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -187,6 +187,14 @@ def courses
     ]
   end
 
+  def custom_button
```
Comment (KevinMulhern) [inline app/helpers/application_helper.rb]: Custom button isn't a very descriptive name, can we change this to something else please?

**Comment by KevinMulhern** [comment]
Hey @mysteryihs, it looks like the master branch is still merged in this PR.

File: app/helpers/button_helper.rb (hunk 1 above)
Comment (KevinMulhern) [inline app/helpers/button_helper.rb]: Indentation is still off here, please take a look at the other methods in this file to see how it should be indented.

Comment (KevinMulhern) [inline app/helpers/button_helper.rb]: Yeah the referrer should be`'home'` instead.

File: app/assets/stylesheets/lessons.scss [hunk 3]
```diff
@@ -0,0 +1,727 @@
+// Place all the styles related to the Lessons controller here.
```
Comment (KevinMulhern) [inline app/assets/stylesheets/lessons.scss]: Can you delete this entire file please? it's in this PR because master is merged in.

**Comment by KevinMulhern** [comment]
Hey @mysteryihs, it looks like master is still merged into this PR. It needs to be merged in eventually I suppose so I'll let is pass. Once you fix the other minor issues, this can be merged 👍 nice work!

File: app/helpers/application_helper.rb (hunk 2 above)
Comment (mysteryihs) [inline app/helpers/application_helper.rb]: I had trouble coming up with a name and this was the best I could think of, what kind of name do you think it should have?

**Comment by mysteryihs** [comment]
I went ahead and removed the ref navs from the button helpers because the buttons aren't always going to come from home or _success_stories so let me know if you want me to put it back.

**Comment by KevinMulhern** [comment]
Thanks @mysteryihs 😄 
This looks great 👍

**Pull request merged**